recipes-overlayed/kselftests/kselftests_4.12-rc7.bb

```diff
@@ -124,20 +130,24 @@ FILES_${PN}-dbg += "${bindir}/kselftests/*/.debug"
 # make[1]: *** [test_verifier] Error 1
 ALLOW_EMPTY_${PN}-bpf = "1"
 
-# FIXME net target builds most of the binaries, but reuseport_bpf_numa depends on libnuma,
-# which is not availbale on ARM, failing entire test case
-ALLOW_EMPTY_${PN}-net = "1"
```

Comment:
have you tried to build an image? while the recipe can build with bitbake kselftests, it can fail at the build image step.